docs/DeveloperGuide.md

```diff
@@ -381,8 +381,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
-* **Private contact detail**: A contact detail that is not meant to be shared with others
 * **DBMS**: Database Management System, a software providing tools for structural data storage.
+* **CLI**: Command-line interface where you interact with the system using your keyboard
+* **API**: Application Programming Interface, a set of programming code that enables data transmission between one software and another
+* **Abstraction and Cohesion**: abstraction is hiding all but relevant data in order to reduce complexity and increase efficiency. Cohesion is the degree which elements belong together
```

Comment:
Minor thing but it would be good to capitalize the first letter for line 387 and 388.